Resources/Locale/en-US/accent/french.ftl

```diff
@@ -0,0 +1,131 @@
+accent-french-replaced-1 = yes
+accent-french-replacement-1 = oui
+
+accent-french-replaced-2 = hi
+accent-french-replacement-2 = salut
+
+accent-french-replaced-3 = hello
+accent-french-replacement-3 = allo
+
+accent-french-replaced-4 = bye
+accent-french-replacement-4 = au revoir
+
+accent-french-replaced-5 = goodbye
+accent-french-replacement-5 = au revoir
+
+accent-french-replaced-6 = the
```

Comment:
This seems excessive. A French speaker would probably use the correct English article, especially since the French article is gendered and the 'le' or 'la' depends on the noun.